lean4-infoview/src/infoview/util.ts

```diff
@@ -325,9 +287,19 @@ export function useLogicalDomObserver(elt: React.RefObject<HTMLElement>): [Logic
 /**
  * An effect which calls `onClickOutside` whenever an element not logically descending from `ld`
  * (see {@link useLogicalDomObserver}) is clicked. Note that `onClickOutside` is not called on clicks
- * on the scrollbar since these should usually not impact the app's state. */
-export function useOnClickOutside(ld: LogicalDomElement, onClickOutside: (_: PointerEvent) => void) {
+ * on the scrollbar since these should usually not impact the app's state.
+ * `isActive` controls whether the `onClickOutside` handler is active. This can be useful for performance,
```

Comment:
Seems reasonable, but I'm not sure whether I like the kind of short lines that occur when consistently using this for independent clauses. E.g. if I am reading the spec correctly, there must be a break before and there should be a break after `This can be useful for performance,`, which would be a relatively short line.

Reply:
'Must be a break' is a bit strong since it's not exactly a mathematically precise spec but yes, if you follow it to the dot you'd probably put a line there. Semantic linebreaking does indeed result in vertical elongation; nevertheless the outputs are far more readable.

Reply:
I'm just echoing the language of rule 4 from the spec you linked :-)